pkg/mount/mount_test.go

```diff
@@ -0,0 +1,139 @@
+package mount
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	mountsConfName    = "mounts.conf"
+	mountsConfContent = "/foo/bar:/foo/bar"
+)
+
+func TestDisableDefaultMountsRevertFunc(t *testing.T) {
```

Comment:
Nit: This and `TestDisableDefaultMounts` would be simpler to follow if not structured as table tests.